### DISCUSSION TITLE
国际服和日服有些问题反馈一下

### DISCUSSION THREAD

**Discussion by HD-906** [MaaAssistantArknights/MaaAssistantArknights#2439] [General]
国际服自动公招，商店和任务都会卡在主界面不动，手动进入也会自动退出来。访问好友时会只访问一个好友就退出来。肉鸽不会投资，默认编队似乎有bug，开局辅助会放弃选择然后会等几秒钟后会放弃本局重开循环。我设置其他初始开局方式后才能恢复正常
日服的话商店和任务都和国际服一样的问题，公招的话只招3个，右下角的槽位不会招。肉鸽有个bug就是如果选了岩溶或带皮肤的白面鸮，那到关卡内会一直重复点击岩溶/白面鸮，不会放下去。

**Comment by renly261** [comment]
我是日服 商店也是卡在主介面不動
